cmd/uql/execute.go

```diff
@@ -143,11 +144,34 @@ func (b defaultBackend) Continue(link *Link) (parsedResponse, error) {
 	}, nil
 }
 
-var backend uqlService = &defaultBackend{}
+type UqlClient interface {
+	ExecuteQuery(query *Query, apiVersion ApiVersion) (*Response, error)
```

Comment:
You have left the public static methods intact. Please, delete them and refactor all uses to a statically provided `UqlClient`